pull

```diff
@@ -89,6 +89,8 @@ echo "Cleaning up Gradle 'buildSrc' scripts"
 rm -f ../buildSrc/src/main/kotlin/force-jacoco.gradle.kts
 rm -f ../buildSrc/src/main/kotlin/deps-between-tasks.kt
 rm -f ../buildSrc/src/main/kotlin/io/spine/internal/gradle/Scripts.kt
+rm -f ../buildSrc/src/main/kotlin/io/spine/internal/gradle/test/Logging.kt
+rm -f ../buildSrc/src/main/kotlin/io/spine/internal/gradle/test/Tasks.kt
```

Comment:
It's a bit fragile. Can we clean everything that's in these folders?

I don't think it's too convenient to remember this place has to be updated for each new file.

Reply:
We can clean the whole `io/spine/internal/gradle/test` folder. But it doesn't work for other folders here. They contain relevant files as well.